### ISSUE TITLE
Gemme adresseopslag

### ISSUE THREAD

**Issue by jma73** [jma73/DtuRunner#2]
Overvej at gemme adresse opslaget til databasen. Således at det kan vises selv når der ikke er internet forbindelse.

Udvid tabel med kolonne til adresse.

Tjekke om adresse allerede er slået op.

Samtidig kunne noget af adressen vises i historik listen.

**Comment by nordfalk** [comment]
Hej, overvejer du at fortsætte udviklingen?
Det ville Henrik sikkert blive glad for

**Comment by jma73** [comment]
Ja, jeg laver en lille smule når jeg har tid. Så må vi se hvor meget det bliver til.
Jeg bruger jo selv app'en en gang i mellem, så det er lidt sjov at arbejde med.
